docs/source-2.0/additional-specs/rules-engine/specification.rst

```diff
@@ -45,8 +46,7 @@ The content of the ``endpointRuleSet`` document has the following properties:
       - Description
     * - version
       - ``string``
-      - **Required**. The rule set schema version. This specification covers
-        version 1.0 of the endpoint rule set.
+      - **Required**. The rules engine version (e.g., 1.0).
```

Comment:
Do we have documentation anywhere of what specifically is supported in each version and what backwards/forwards compatibility considerations are?

Reply:
I added a simple section about the version number.